### ISSUE TITLE
Questions about model training

### ISSUE THREAD

**Issue by chunweihuangya** [cqylunlun/GLASS#83]
Have you tried using early stopping and a learning rate scheduler when training the model?

**Comment by cqylunlun** [comment]
We have not yet attempted early stopping strategies. Instead, we followed a common paradigm from [SimpleNet](https://github.com/DonaldRR/SimpleNet/blob/351a2b8d4e8cfc944dbccbf9bc6ceda930c6f26b/simplenet.py#L419-L444) used in anomaly detection tasks. Since the dataset contains relatively few anomalous samples and no separate validation set is provided, the test set is directly used as the validation set.

**Comment by chunweihuangya** [comment]
Another question about model training: Why do you use "image_auroc + pixel_auroc" to evaluate model performance
